app/src/components/common/names-capture.vue

```diff
@@ -1017,8 +1017,8 @@ export default class NamesCapture extends Mixins(CommonMixin) {
 
 .launch-icon {
   display: inline-block;
-  font-size: $px-14;
-  color: $app-blue;
+  font-size: $px-14 !important;
+  color: $app-blue !important;
```

Comment:
I didn't dig into why the previous styles were being overridden, but this now displays correctly.